plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/featured.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import type {
+	BlockAttributes,
+	InnerBlockTemplate,
+	BlockIcon,
+} from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { Icon, starFilled } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DEFAULT_ATTRIBUTES,
+	INNER_BLOCKS_PRODUCT_TEMPLATE,
+} from '../constants';
+import { CoreCollectionNames, CoreFilterNames } from '../types';
+
+const collection = {
+	name: CoreCollectionNames.FEATURED,
+	title: __( 'Featured', 'woocommerce' ),
+	icon: ( <Icon icon={ starFilled } /> ) as BlockIcon,
+	description: __( 'Showcase your featured products.', 'woocommerce' ),
+	keywords: [],
```

Comment:
I agree and wouldn't mind removing it since that property is optional and defaults to an empty array. But I already added a `product collection` keyword to collections so it no longer applies 😄